cpp/RNSScreenRemovalListener.cpp

```diff
@@ -0,0 +1,25 @@
+#include "RNSScreenRemovalListener.h"
+#include <react/renderer/mounting/ShadowViewMutation.h>
+using namespace facebook::react;
+
+std::optional<MountingTransaction> RNSScreenRemovalListener::pullTransaction(
+    SurfaceId surfaceId,
+    MountingTransaction::Number transactionNumber,
+    const TransactionTelemetry &telemetry,
+    ShadowViewMutationList mutations) const {
+  for (ShadowViewMutation mutation : mutations) {
```

Comment:
```suggestion
  for (const ShadowViewMutation& mutation : mutations) {
```
no need for excessive copying here. This is not a big type, however we might have a lot of mutations here.